apps-devstg/us-east-1/aws-cloudwatch-synthetics/s3.tf

```diff
@@ -0,0 +1,13 @@
+module "target_canary_s3_bucket" {
+
+  source = "github.com/binbashar/terraform-aws-s3-bucket.git?ref=v3.14.0"
+
+  bucket = "${var.project}-${var.environment}-target-canary"
+
+  force_destroy = true
+
+  control_object_ownership = true
+  object_ownership         = "BucketOwnerEnforced"
```

Comment:
Just in case, the best practice is to disable ACLs unless you really know what you are doing.

Reply:
@diego-ojeda-binbash as far as I underdstand, setting the ownership sets it to "ACLs are disabled. All objects in this bucket are owned by this account. Access to this bucket and its objects is specified using only policies.", am I right?

Reply:
I can't tell but I'll trust that's what you saw when you created the bucket. 👍